Testscripts/Windows/VERIFY-HPC-VM.ps1

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This PS1 is not required. Framework has a support the Linux scripts directly and collect the state file and log files.
Remove this PS1 file.

Reply:
As I said to Lili, I may have one more bash script independently later. I will keep this form.

Reply:
Can we add at that point of time when we enhance ?